andela/coding-geeks-ah-frontend
# #167484559 implemented view notifications feature

#### What does this PR do?
It implements the view notifications feature.

#### What to check
1. Connect the client socket to the localhost backend to see if the socket status connection is true.
2. Connect the client socket to the Heroku backend to see if the socket status connection is false.

#### How should this be manually tested?
1. Clone the branch.
2. Do **npm install**.
3. Do **npm start**.
4. Go the **routes** subfolder and open the **NavBar.js** file.
5. Change the endpoint state accordingly by switching between the localhost and Heroku backend to see the connection status.

#### Screenshots
**Local Host: Connected = True**

<img width="1421" alt="Screen Shot 2019-11-11 at 14 31 30" src="https://user-images.githubusercontent.com/34879554/68587644-3c330600-0490-11ea-9dff-ff4d1c973d3c.png">
<img width="508" alt="Screen Shot 2019-11-11 at 14 31 50" src="https://user-images.githubusercontent.com/34879554/68587646-3ccb9c80-0490-11ea-90cf-578aea0a0b51.png">

**Heroku: Connected = False**
<img width="1415" alt="Screen Shot 2019-11-11 at 14 32 32" src="https://user-images.githubusercontent.com/34879554/68587684-50770300-0490-11ea-96cd-b92411a332be.png">
<img width="689" alt="Screen Shot 2019-11-11 at 14 33 55" src="https://user-images.githubusercontent.com/34879554/68587687-510f9980-0490-11ea-857b-20e8a8445af0.png">
